src/main/java/frc/robot/RobotContainer.java

```diff
@@ -69,6 +72,8 @@ private void configureButtonBindings() {
     OI.Driver.getAlignBackPOV().onTrue(new InstantCommand(() -> swerveSubsystem.enableRotationHold(180), swerveSubsystem));
     OI.Driver.getAlignLeftPOV().onTrue(new InstantCommand(() -> swerveSubsystem.enableRotationHold(90), swerveSubsystem));
     OI.Driver.getAlignRightPOV().onTrue(new InstantCommand(() -> swerveSubsystem.enableRotationHold(-90), swerveSubsystem));
+    OI.Operator.deployBuddyBalanceButton().onTrue(new InstantCommand(buddyBalanceSubsystem::deployBuddyBalance));
+    OI.Operator.retrieveBuddyBalanceButton().onTrue(new InstantCommand(buddyBalanceSubsystem::retrieveBuddy));
```

Comment:
```suggestion
    OI.Operator.deployBuddyBalanceButton().onTrue(new InstantCommand(buddyBalanceSubsystem::deployBuddyBalance, buddyBalanceSubsystem));
    OI.Operator.retrieveBuddyBalanceButton().onTrue(new InstantCommand(buddyBalanceSubsystem::retrieveBuddy, buddyBalanceSubsystem));
```
Need to add subsystem requirements to the InstantCommand